### PR TITLE
Add NeosLinuxExportFix

### PR DESCRIPTION
**Pull request by Frozenreflex**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3082,6 +3082,33 @@
...
+            "sourceLocation": "https://github.com/Frozenreflex/NeosLinuxExportFix",
+            "authors": {
+                "Fro Zen": {
+                    "url": "https://github.com/Frozenreflex"
+                }
+            },
+            "flags": [
+                "broken:linux-wine",
+                "broken:windows"
+            ],
```
Comment (EIA485) [inline manifest.json:3098]: these need to be version flags [docs/manifest-flags](https://www.neosmodloader.com/manifest-flags)

Comment (Frozenreflex) [reply manifest.json:3098]: https://discord.com/channels/901126079857692714/1011888035924803634/1067251932596555878

Comment (EIA485) [reply manifest.json:3098]: alright then, il create some prs to update our docs/schema definition

**Comment by Banane9** [comment]
@Frozenreflex your broken flags need to go onto the individual versions of the mod, not on the mod itself (even if that'd make more sense considering what it is)

**Comment by Frozenreflex** [comment]
> @Frozenreflex your broken flags need to go onto the individual versions of the mod, not on the mod itself (even if that'd make more sense considering what it is)

did you read the resolved conversation

**Review by zkxs — APPROVED** [review]
Looks good. Sorry for the wait... had to change the schema.json for this PR.

random nonsense thoughts
- semaphores are scary, nice to see someone use them correctly
- your readme is funny

**Pull request merged**